buildtest/cli/report.py

```diff
@@ -600,6 +600,15 @@ def get_buildspecs(self):
         """Return a list of buildspecs in report file"""
         return self.filtered_buildspecs
 
+    def get_failed_tests(self):
+        """Return a list of failed test names from report file"""
+        test_names = []
```

Comment:
I think to be safe you should use a set i.e `test_names = set()` and then add items `test_names.add(name)` to avoid adding duplicate test and the return can be a list i.e `list(test_names)`

Reply:
Make sense!!